src/platform/silabs/wifi/SiWx/WifiInterface.cpp

```diff
@@ -408,6 +413,42 @@ sl_status_t SetWifiConfigurations()
     return status;
 }
 
+/**
+ * @brief Callback function for the SL_WIFI_JOIN_EVENTS group
+ *
+ * This callback handler will be invoked when any event within join event group occurs, providing the event details and any associated data
```

Comment:
Callback doesn't get called we joint a network when using the sl net APIs. comment should be updated.